content/reference/endpoints/bookings.md

```diff
@@ -59,41 +59,43 @@ payments          | Array    | Read      | Array of related Payments IDs |
-external_reference      | String  | Read/Write | Booking's reference from the source channel. |
-cancelation_reason      | String  | Read/Write | Booking's cancelation reason (for writable part - it can be used in both update and cancelation endpoints) |
-cancelation_message_to_guest      | String  | Read/Write | Booking's cancelation message to guest (for writable part - only in cancelation endpoint) | 
-cancelation_message_to_channel    | String  | Read/Write | Booking's cancelation message to guest (for writable part - only in cancelation endpoint) |
-imported                | Boolean  | Read/Write | Whether this booking was imported from the external Channel or not | 
-reconciliation          | [Decimal](/reference/enums#formats  | Read/Write | Booking's reconciliation | 
-authorized_amount       | [Decimal](/reference/enums#formats  | Read | Booking's authorized amount | 
-paid_or_authorized_amount | [Decimal](/reference/enums#formats  | Read | Booking's paid or authorized amount |
+id                                             | Integer | Read       | Booking's id. |
+client_id                                      | Integer | Write      | Client id related to the Booking. |
+source_id                                      | Integer | Write      | Source id related to the Booking. |
+bookings_tag_ids                               | Array   | Write      | Array of ids of BookingsTags to be associated with Booking. |
+adults                                         | Integer | Read/Write      | Booking's number of adults. |
+booked                                         | Boolean | Read/Write      | Booking's booked status, false by default. When set to true, marks a regular booking. |
+channel_price                                  | Integer | Read/Write      | Price accepted by the traveler on the channel site, excluding traveler's fees (eg: Airbnb). |
+charge_damage_deposit_on_arrival               | Boolean    | Read/Write | Defines if charge damage deposit will be charged on arrival. | Can be `true` or `false` for Pro subscriptions, only `false` allowed when `damage_deposit_collection_method` is `"smily_damage_deposit_with_swikly"` |
+children                                       | Integer | Read/Write      | Booking's number of children. |
+commission                                     | [Decimal](/reference/enums#formats) | Write | Booking's commission fee. | Requires `final_price` to be present
+currency                                       | String  | Read/Write      | Booking's currency code, list of codes described in [enums section](/reference/enums#currencies). | Must be blank or included in [currencies list](/reference/enums#currencies)
+damage_deposit                                 | [Decimal](/reference/enums#formats) | Read/Write | Booking's damage deposit. | greater than or equal to: 0 |
```

Comment:
Maybe we can add that it is readonly and autogenerated in case of `smily_damage_deposit_with_swikly`?

Reply:
Mentioned it, but only there where is the "write" scope